coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -70,6 +71,24 @@ func TestUpdateMetrics_MetricsDoNotExpire(t *testing.T) {
 			{Name: "hello", Value: "world"},
 		}},
 		{Name: "d_gauge_four", Type: agentproto.Stats_Metric_GAUGE, Value: 6},
+		{Name: "e_gauge_four", Type: agentproto.Stats_Metric_GAUGE, Value: 15, Labels: []*agentproto.Stats_Metric_Label{
+			{Name: "foobar", Value: "Foo,ba=z"},
+			{Name: "hello", Value: "wo,,r=d"},
+		}},
+		{Name: "f_gauge_four", Type: agentproto.Stats_Metric_GAUGE, Value: 6, Labels: []*agentproto.Stats_Metric_Label{
+			{Name: "foobar", Value: "foobaz"},
+			{Name: "empty", Value: ""},
```

Comment:
I think these need to be in lexicographical order

Reply:
Fixed the order.